test/functional/cmdLineTests/shareClassTests/SCCMLTests/playlist.xml

```diff
@@ -206,12 +206,13 @@
 		</variations>
 		<command>cp $(Q)$(TEST_RESROOT)$(D)..$(D)URLHelperTests$(D)URLHelperTests.jar$(Q) .; \
 	true URLHelperTests.jar ;\
+	cp $(Q)$(JVM_TEST_ROOT)$(D)functional$(D)cmdLineTests$(D)utils$(D)utils.jar$(Q) $(Q)$(JVM_TEST_ROOT)$(D)functional$(D)cmdLineTests$(D)utils$(D)utils$(Q); \
```

Comment:
This is only needed for the new test cases added in the PR. Instead of doing the cp here, can this be moved above the 200a test case in ShareClassesCMLTests-5.xml. We have existing examples of running preparation steps using `<exec command=...>` in CML tests.